server/src/main/java/org/apache/druid/curator/CuratorConfig.java

```diff
@@ -26,16 +26,20 @@
 
 import javax.validation.constraints.Min;
 
-/**
- */
 public class CuratorConfig
 {
-  @JsonProperty("host")
+  static final String HOST = "host";
```

Comment:
nit: if string literal is only used once then creating a static variable for it is overkill IMO.

Reply:
It's also used CuratorModuleTest.java (I added the constant to keep the usages in sync).